mbedtls-sys/Cargo.toml

```diff
@@ -50,3 +50,4 @@ aesni = []
 padlock = []
 legacy_protocols = []
 mpi_force_c_code = []
+trusted_cert_callback = []
```

Comment:
It seems like enabling this feature in mbedtls adds very little additional code size. Could we just enable it always to avoid further proliferation of feature flags?

Reply:
It's correct that this is a feature on mbedtls-sys. It could be turned by default in mbedtls though.

Reply:
Indeed, I've update the `Cargo.toml` in the mbedtls crate to enable the `trusted_cert_callback` feature in `mbedtls-sys-auto` by default.